packages/wallet/contract/src/smartWallet.js

```diff
@@ -1,90 +1,425 @@
 // @ts-check
+import { AmountMath } from '@agoric/ertp';
+import { isNat } from '@agoric/nat';
 import {
   makePublishKit,
   makeStoredSubscriber,
   observeIteration,
 } from '@agoric/notifier';
-import { E, Far } from '@endo/far';
-import { makeWallet } from './support.js';
+import { makeScalarWeakMapStore } from '@agoric/store';
+import { makeScalarBigMapStore } from '@agoric/vat-data';
+import { E, Far, passStyleOf } from '@endo/far';
+import { makeInvitationsHelper } from './invitations.js';
+import { paymentsFromPurses, paymentsIntoPurses } from './utils.js';
 
-const { assign, entries, keys, fromEntries } = Object;
+/**
+ * @file Smart wallet module
+ *
+ * Design notes:
+ * PRIME DIRECTIVE: DON'T LOSE MONEY.
```

Comment:
Nit: "assets" or "tokens" instead of money

Reply:
removed